### ISSUE TITLE
java.lang.IllegalArgumentException: Value too long

### ISSUE THREAD

**Issue by theranger969** [TurboVNC/turbovnc#191]
Hello, I just upgraded to TurboVNC v2.2.2 on my Mac Mojave v10.14.6.  Since I have upgraded I am receiving the error:

"java.lang.IllegalArgumentException: Value too long: hostname, hostname, hostname, IP address, hostname...."

It seems to list every hostname I've connected to.  It will occur on any given hostname or IP address at any given time.  But then there are computers I am able to connect to with no problem.  I have Java 8 Update 221 installed.

Not sure if there is a cache file that needs deleted or what else to do.  I tried searching to see if there was another update or fix but was unable to locate anything.

Also I am not that great at programming so please bare with me.  Any further information please let me know and thank you for any assistance.

**Comment by dcommander** [comment]
Try the 2.2.x stable pre-release build:
https://turbovnc.org/DeveloperInfo/PreReleases
If the problem still occurs, then I'll look into it.

**Comment by theranger969** [comment]
Thank you for your reply.  I have downloaded v2.2.1 and installed it.  I get the same thing as described above.  Would you like me to try another earlier version? Thank you for your help!

**Comment by dcommander** [comment]
I didn't say 2.2.1.  I said the 2.2.x pre-release build, which is linked above.  That build will become 2.2.3 eventually.  Please follow instructions.

**Comment by theranger969** [comment]
I removed the previous version and installed that version.  I get the same as above.  Sorry for the confusion.

**Comment by dcommander** [comment]
I can't reproduce the issue.  I would suggest the following:

1. Open the TurboVNC Viewer.
2. Open the Options dialog.
3. Go to the "Global" tab.
4. Click "Clear the list of saved connections."
5. Cancel the dialog.
6. Quit and restart the TurboVNC Viewer.

If the issue still occurs after that, then please give me some examples of hostnames/IP addresses that fail.

**Comment by theranger969** [comment]
Thank you dcommander.  I had tried this before and it did not work.

But for whatever reason, I cleared it again, uninstalled the viewer, reinstalled it, cleared the list again before I did anything else, closed out of the viewer and then reopened.  Now I can connect to all the hostnames/IP addresses that I was having problems with.  Not sure what the world happened.  Anyway, thank you for your suggestions and help!  This made my day and am very appreciative!
